docs/variants/novacustom_v540tu/test-matrix.md

```diff
@@ -0,0 +1,137 @@
+# Test matrix
+
+## About
+
+The test matrix is used to determine the scope of tests which the DUT is
+subjected from before the release of the new binary.
```

Comment:
@mkopec, @macpijan 
I am wondering if this is the correct way to provide a test matrix, as this list tends to become outdated pretty quickly, since we are constantly adding new test cases. Maybe linking to the  [Dasharo Test & Feature Matrix](https://docs.google.com/spreadsheets/d/1wSE6xA3K3nXewwLn5lV39_2wZL1kg5AkGb4mvmG3bwE/edit#gid=736501945) could be sufficient if we could provide the overall test scope for one platform (not a given release) in a sheet? WDYT?

Reply:
Definitely, makes sense, though maybe we should add a new summary page that shows a list of tests enabled for a given platform

Reply:
done in commit 613858b9c6759a31ecc8f486fc8f7230ad1fc5f9